Release Notes + Highlights for 2.11.1 (#7510)
elastic/cloud-on-k8s: docs/release-notes/2.11.0.asciidoc

[float]
=== Breaking changes

> * The `resourceStatuses` field of the status subresource of the Stack Configuration Policy is no longer in use. Instead a new `details` field is populated which now also contains information about configured Kibana applications. This change could cause errors during an upgrade of the CRDs while your operator is still running an older version but 2.11.1 was released to mitigate this issue. {pull}7433[#7433]

Reviewer: nit:

```suggestion
* The `resourceStatuses` field of the status subresource of the Stack Configuration Policy is no longer in use. Instead a new `details` field is populated which now also contains information about configured Kibana applications. This change could cause errors during an upgrade of the CRDs while your operator is still running an older version, 2.11.1 has been released to mitigate this issue. {pull}7433[#7433]
```

(not a native speaker, feel free to ignore 😄 )

Author: > (not a native speaker, feel free to ignore 😄 )

Nope, I see your point here, but I added a `however` after the comma, as it made it flow a bit better.
